packages/teams-js/src/public/constants.ts

```diff
@@ -82,3 +82,18 @@ export enum ChannelType {
 }
 
 export const errorNotSupportedOnPlatform: SdkError = { errorCode: ErrorCode.NOT_SUPPORTED_ON_PLATFORM };
+
+export const minRuntimeConfigToUninitialize = {
```

Comment:
Is this only used for unit tests? If so, I think we should prefix it with "_" (like we do for `app._initialize`, etc.) and add an @hidden tag, with a comment explaining it's only for unit tests, so it's not included in the public docs.

Reply:
Sure, will make the changes accordingly.

Reply:
Also please add an @internal tag. :)

Reply:
I'm also wondering if this shouldn't go in the runtime.ts file, rather than constants.ts

Reply:
I thought of putting all the constants in contants.ts but will move to runtime.ts